src/main/java/uk/gov/hmcts/reform/hmc/api/config/ServiceBusConfiguration.java

```diff
@@ -139,8 +149,19 @@ public CompletableFuture<Void> onMessageAsync(IMessage message) {
                                             + hearingDto.getHearingUpdate().getHearingVenueName());
                         }
 
+                        Hearings hearings =
+                                hearingsService.getHearingsByCaseRefNo(hearingDto.getCaseRef());
+
+                        if (hearings != null) {
+                            nextHearingDetailsService.updateNextHearingDate(hearings);
+                        }
+
+                        State caseState =
+                                nextHearingDetailsService.fetchStateForUpdate(
```

Comment:
will this code work if hearings is null?

Reply:
I wont work , will throw null pointer exception, . Now resolved